TLM/TLM/UI/SubTools/LaneArrowTool.cs

```diff
@@ -104,7 +123,14 @@ public LaneArrowTool(TrafficManagerTool mainTool)
             }
 
             int width = numLanes * 128;
-            var windowRect3 = new Rect(screenPos.x - (width / 2), screenPos.y - 70, width, 50);
+            int height = 50;
+            bool startNode = (bool)netService.IsStartNode(SelectedSegmentId, SelectedNodeId);
+            if (CanReset(SelectedSegmentId, startNode)) {
+                Log._Debug("Expanding height");
```

Comment:
Logging in GUI update call is not the best idea

Reply:
Doesn't `_Debug()` get auto-pruned from `RELEASE` builds?

Reply:
Do you really want thousands of useless lines in your log while running debug build?

Reply:
Uhm actually, will make `DEBUG` builds slower to test. Isn't there a logging switch for Speed Limits? _checks...._

[Logging docs](https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition/blob/master/docs/LOGGING.md)

[Loggin switches](https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition/blob/master/TLM/TLM/State/ConfigData/DebugSettings.cs#L17-L45) seem to contain some that might be relevant:

```cs
false, // 1: routing basic debug log
false, // 8: debug routing
false, // 23: debug lane connections
```

Reply:
Tell me how useful that info is for you? For me it's useless (pure visual thing), because if _CanReset_, height will be expanded, but _Reset_ button might become rendered or not (_see GuiLaneChangeWindow_ - it's tested again before actual rendering)

Regards to __multiple times per frame__ see [MonoBehaviour.OnGUI](https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnGUI.html)

Reply:
Opps forgot to remove it!